updateVersions.sc

```diff
@@ -123,7 +123,7 @@ def main(updatedImage: String, updatedImageReleaseNote: String): Unit = {
                 val firstSplit = s.split(":")
                 val splited = firstSplit(1).split("\\.")
                 s"${firstSplit(0)}:${splited(0)}.${splited(1)}.${splited(2).toInt + 1}\n"
-              } else if(s.contains(" AS ")) {
+              } else if(s.contains(" AS ") && imagesToUpdate.contains("terra-jupyter-python")) {
```

Comment:
Just thinking through.. this works because we do:
```
FROM r-image
FROM py-image AS python
```
right? So we will only replace the second line if the py image is actually being updated.

This would break if we used "AS" in other ways, like
```
FROM some-other-image AS foo
```

Maybe could change it to:
```
else if (s.endsWith(" AS python") && imagesToUpdate.contains("terra-jupyter-python"))
```
to be a bit more robust?

Reply:
correct - it would break if we used anything but `python` after `AS`. I'll update now!

Reply:
does this work if we only want to update R?

Reply:
I think that's what this fixes -- now the else if clause won't pass and we won't mistakenly update python.

Reply:
yeah I tested with `amm ./updateVersions.sc terra-jupyter-r "[message]"` and the versions bumps looks good as seen in this PR https://github.com/DataBiosphere/terra-docker/pull/184/files

Reply:
oh nvm..I thought we have `AS R` for `FROM us.gcr.io/broad-dsp-gcr-public/terra-jupyter-r:??? ` as well